front-end/store/modules/SettingStore.js

```diff
@@ -23,6 +30,10 @@ const actions = {
     [SETTINGS_TOGGLE_NOTIFICATIONS]({ commit, state }) {
         commit(state.pushNotifications ? SETTINGS_SET_NOTIFICATIONS_OFF : SETTINGS_SET_NOTIFICATIONS_ON);
     },
+
+    [SETTINGS_SET_NOTIFICATION_STATUSES]({ commit, state }) {
```

Comment:
This action is not needed as it only proxies to the mutation. You can directly call a mutation by `this.$store.commit(SETTINGS_SET_NOTIFICATION_STATUSES, statuses)` in your vue objects. Please remove this one :)

Reply:
Ah you're already using `commit`, than this function doesn't do anything at all. Please remove it :)